portfolio/src/main/webapp/index.html

```diff
@@ -70,7 +70,7 @@ <h2>Comments</h2>
     
     <h2> Fetch comment information </h2>
       <p>Click here to get comment info:</p>
-     <ul id="Comments"></ul>
+     <ul id="Comments"></ul> <!-- make div and then make seperate boxes of p to look more like comments see BS documentation-->
```

Comment:
Is this a TODO? Make sure to add TODO in the comment. The convention at Google for C++ is usually TODO(kkincy).